### PR TITLE
Wagner

### PR DESCRIPTION
**Pull request by Mach131**

### CODE REVIEW

File: test/unit/test_shovelshield.gd [hunk 1]
```diff
@@ -367,3 +367,19 @@ func test_shovelshield_inseperable_where_are_you_going():
...
+	player1.on_position_changed(3, 3, false)
+
+	give_player_specific_card(player1, "shovelshield_tandemattack", TestCardId3)
+	assert_true(game_logic.do_boost(player1, TestCardId3, [player1.hand[0].id]))
+
+	give_player_specific_card(player2, "standard_normal_dive", TestCardId4)
+	assert_true(game_logic.do_boost(player2, TestCardId4, []))
+	assert_true(game_logic.do_boost_name_card_choice_effect(player2, TestCardId3))
+
+	execute_strike(player1, player2, "standard_normal_sweep","standard_normal_grasp", [], [1], false, false)
```
Comment (daniel-k-taylor) [inline test/unit/test_shovelshield.gd:383]: can you humor me and do another test where you don't push them off the buddy and make sure that aspect still doesn't trigger?

File: data/card_definitions.json [hunk 2]
```diff
@@ -21639,6 +21639,363 @@
...
+		"type": "ultra",
+		"display_name": "Hitze Falke",
+		"force_cost": 0,
+		"gauge_cost": 2,
+		"range_min": 3,
+		"range_max": 4,
+		"speed": 1,
+		"power": 7,
+		"armor": 3,
+		"guard": 3,
```
[the remark this answers is not in the document]
Comment (Mach131) [reply data/card_definitions.json:21694]: i think it's not actually

File: scenes/game/game.gd [hunk 3]
```diff
@@ -894,6 +894,10 @@ func can_select_card(card):
 	var in_choice_zone = is_card_in_player_reference($AllCards/ChoiceZone.get_children(), card.card_id)
 
 	if ui_state == UIState.UIState_PickTurnAction:
+		if in_player_boosts:if in_player_boosts:
```
[the remark this answers is not in the document]
Comment (Mach131) [reply scenes/game/game.gd:897]: o whoops

File: test/unit/test_shovelshield.gd (hunk 1 above)
Comment (Mach131) [reply test/unit/test_shovelshield.gd:383]: this one was also funny because they ended up with -3power

**Review by daniel-k-taylor — APPROVED** [review]

**Pull request merged**